### PR TITLE
 added ./idea/ in the .gitignore

### PR DESCRIPTION
**Pull request by ibhavikmakwana**
added this file to your .gitignore because these files can contains some contents that is not necessary like absolute path.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,4 @@ google-services.json
 freeline.py
 freeline/
 freeline_project_description.json
+!/.idea/
```
Comment (Bloody-Badboy) [inline .gitignore]: whole `.idea/` is not excluded because it contents the project code styles

Comment (ibhavikmakwana) [reply .gitignore]: Ohk get it,
What about the "caches", it should be in the .gitignore?

Comment (Bloody-Badboy) [reply .gitignore]: Already Update that @See develop branch https://github.com/UdacityAndroidDevScholarship/blood-donation/blob/f193c9f5eb733b41358565c3d0d50a142d95835a/.gitignore#L42

**Comment by sakchhams** [comment]
closing as per above discussion.

**Pull request closed without merging**